test/screenshot/screenshot.tsx

```diff
@@ -90,10 +98,16 @@ export default class Screenshot {
         return;
       }
       // Save the snapshot and the diff
-      await Promise.all([
-        this.saveImage_(snapshotPath, snapshot, metadata),
-        this.saveImage_(diffPath, diff, metadata),
-      ]);
+      if (storage) {
+        await Promise.all([
+          this.saveImage_(snapshotPath, snapshot, metadata),
+          this.saveImage_(diffPath, diff, metadata),
+        ]);
+      }
+      if (Number(data.misMatchPercentage) > 0) {
+        mkdirp.sync(path.dirname('no_match/' + diffPath));
```

Comment:
please make `no_match/` a constant at top of file.

Also prefer using back ticks 

```
`${NO_MATCH_DIRECTORY}${diffPath}`
```